File: OLSource1/v1/VS_officedev/codesnippet/VisualBasic/associating-a-form-region-with-an-outlook-message-class_1.vb
    <Microsoft.Office.Tools.Outlook.FormRegionMessageClass(Microsoft.Office.Tools.Outlook.FormRegionMessageClassAttribute.Note)> _
    <Microsoft.Office.Tools.Outlook.FormRegionMessageClass("IPM.Task.Contoso")> _
    <Microsoft.Office.Tools.Outlook.FormRegionName("Trin_Outlook_FR_Attributes.FormRegion1")> _
    Partial Public Class FormRegion1Factory

    ' Occurs before the form region is initialized.
    ' To prevent the form region from appearing, set e.Cancel to true.
    ' Use e.OutlookItem to get a reference to the current Outlook item.
        Private Sub FormRegion1Factory_FormRegionInitializing(ByVal sender As Object, ByVal e As Microsoft.Office.Tools.Outlook.FormRegionInitializingEventArgs) Handles Me.FormRegionInitializing

    End Sub

File: OLSource1/v1/VS_officedev/codesnippet/CSharp/associating-a-form-region-with-an-outlook-message-class_1.cs
        [Microsoft.Office.Tools.Outlook.FormRegionMessageClass
            (Microsoft.Office.Tools.Outlook.FormRegionMessageClassAttribute.Note)]
        [Microsoft.Office.Tools.Outlook.FormRegionMessageClass
            ("IPM.Task.Contoso")]
        [Microsoft.Office.Tools.Outlook.FormRegionName
            ("Trin_Outlook_FR_Attributes.FormRegion1")]
        public partial class FormRegion1Factory
        {
            // Occurs before the form region is initialized.
            // To prevent the form region from appearing, set e.Cancel to true.
            // Use e.OutlookItem to get a reference to the current Outlook item.
            private void FormRegion1Factory_FormRegionInitializing(object sender, 
                Microsoft.Office.Tools.Outlook.FormRegionInitializingEventArgs e)
            {
            }
        }